tests/virt_autotest/update_package.pm

```diff
@@ -31,21 +31,28 @@ sub update_package {
     }
 
     $update_pkg_cmd = $update_pkg_cmd . " 2>&1 | tee /tmp/update_virt_rpms.log ";
-    $ret            = $self->execute_script_run($update_pkg_cmd, 7200);
+    if (check_var('ARCH', 's390x')) {
+        type_string("$update_pkg_cmd\n");
+        upload_asset"/tmp/update_virt_rpms.log", 1, 1;
```

Comment:
How to handle the update_virt_rpms failures? Seems missing. So does other cmds. Do not we need to consider whether commands finish as expected?